aip-console-jenkins/src/main/java/io/jenkins/plugins/aipconsole/SnapshotBuilder.java

```diff
@@ -234,6 +237,37 @@ public void perform(@Nonnull Run<?, ?> run, @Nonnull FilePath workspace, @Nonnul
                 resolveSnapshotName = String.format("Snapshot-%s", new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS").format(new Date()));
             }
 
+            //TODO: refactor after release to get separated workflows
+            if (app.isOnboarded()) {
+                log.println(SnapshotBuilder_Snapshot_info_fastScanWorkflow(expandedAppName));
+                ScanAndReScanApplicationJobRequest.ScanAndReScanApplicationJobRequestBuilder requestBuilder = ScanAndReScanApplicationJobRequest.builder()
+                        .appGuid(applicationGuid);
+                String targetNode = app.getTargetNode();
+                if (StringUtils.isNotEmpty(targetNode)) {
+                    requestBuilder.targetNode(targetNode);
+                }
+
+                if (StringUtils.isNotEmpty(caipVersion)) {
+                    requestBuilder.caipVersion(caipVersion);
+                }
+                if (StringUtils.isNotEmpty(resolveSnapshotName)) {
+                    requestBuilder.snapshotName(resolveSnapshotName);
+                }
+
+                requestBuilder.processImaging(processImaging);
+                requestBuilder.publishToEngineering(processImaging || consolidation);
+                requestBuilder.uploadApplication(true);
```

Comment:
why is the uploadApplication always true ? this is to send application in measure, and for that you need the publishToEngineering

Reply:
So, instead, it should be conditioned to consolidation.
Before you ask, consolidation is an existing parameter that allows uploading data after snapshots are taken.
As you might be aware, you are in the snapshotBuilder to trigger snapshot

Reply:
If you are in the snapshot builder, why do you process imaging then ?

Reply:
How to say that,?
You're publishing after the snapshot was taken.
You can choose not to publish